src/Components/ScrivitoExtensions/IconEditorTab/AllIcons.js

```diff
@@ -12,42 +12,29 @@ fontAwesomeIcons.forEach((icon) =>
   })
 );
 
-class AllIcons extends React.Component {
-  constructor(props) {
-    super(props);
+function AllIcons({ setWidgetIcon, currentIcon, hide }) {
+  const [initialRender, setInitialRender] = React.useState(true);
+  React.useEffect(() => {
+    setTimeout(() => setInitialRender(false), 10);
```

Comment:
* [ ] FYI: reading from the top I thought "Maybe the `Note: the initialRender is a performance tweak` comment should go here".